client.go

```diff
@@ -297,6 +297,24 @@ func (c *Client) GetFile(u string, params url.Values) (*FileResponse, error) {
 	}, nil
 }
 
+// ReadSeekCloser implements all three of Seeker, Closer and Reader interfaces
+type ReadSeekCloser interface {
+	io.ReadSeeker
+	io.Closer
+}
+
+// OpenFile opens file using HTTP protocol and uses `Range` headers
+// to Seek to various positions in the file, this means that server
```

Comment:
Seek -> seek
And it's not totally clear which flags server needs to support as if the comment is not complete